Umbraco-Cloud/Set-Up/Media/index.md

```diff
@@ -30,15 +30,15 @@ When you add new media files to your project while working on a local clone, the
 
 ## One Storage per Environment
 
-Each environment on your Umbraco Cloud project has a separate Azure Blob Storage container. Each of your environments use a storage container specific to that environment.
+Each environment on your Umbraco Cloud project has a separate Azure Blob Storage container. Each of your environments uses a storage container specific to that environment.
 
 When you deploy between two environments on your Umbraco Cloud project the media files will be copied from one storage to the other, depending on which environment is being transferred to and from.
 
 As an example, imagine that you are transferring new content changes from your Development environment to your Live environment. When you initiate the transfer, all media files from the Azure Blog Storage container connected to your Development environment will be copied and pasted into the container connected to your Live environment. Once all content changes have also been transferred, and the transfer is complete, your Media libraries on the two environments will now be in sync.
 
 ## Media Folder
 
-You will notice that there is a Media (`/media`) folder in the root of your project files. This folder usually holds all media files associated with a Umbraco project. As your Umbraco Cloud environments are all configured with Azure Blob Storage, the media files are not in this media folder.
+You will notice that there is a Media (`/media`) folder at the root of your project files. This folder usually holds all media files associated with a Umbraco project. As your Umbraco Cloud environments are all configured with Azure Blob Storage, the media files are not in this media folder.
```

Comment:
I think this should be clarified a bit more, as the media folder is in the wwwroot/media folder and not just in the root of the project :)

Reply:
on v9 and 10 at least ;)